lib/apple/index.js

```diff
@@ -32,14 +46,27 @@ function parseResult(result) {
 		throw error;
 	}
 
-	var receipt = result.receipt;
+	var productId = getReceiptFieldValue(result.receipt, 'product_id');
+	var transactionId = getReceiptFieldValue(result.receipt, 'transaction_id');
+
+	/* jshint camelcase:false */
+    if (result.hasOwnProperty('latest_receipt_info') && result.latest_receipt_info[0]) {
+        latestReceiptInfo = result.latest_receipt_info.sort(function (a, b) {
+            a = parseInt(a.original_purchase_date_ms);
+            b = parseInt(b.original_purchase_date_ms);
+
+            return a - b;
+        });
+
+        productId = latestReceiptInfo.slice(-1).pop().product_id;
+        transactionId = latestReceiptInfo.slice(-1).pop().transaction_id;
```

Comment:
Sorry, I'm a bit sensitive to performance nits. Would you mind reading from `latestReceiptInfo[latestReceiptInfo.length - 1]` instead?